### PR TITLE
Docs: Adding panel suggestions initial documentation for 8.3

### PR DESCRIPTION
**Pull request by pkolyvas**
This PR contains minor changes to the "Add a panel" & "Visualizations" documentation to include the new panel suggestions feature shipping in 8.3.

### CODE REVIEW

File: docs/sources/panels/add-a-panel.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ Each panel needs at least one query to display a visualization. You write querie
...
 
-## 3. Choose a visualization type
-
-In the Visualization list, click a visualization type. Grafana displays a preview of your query results with that visualization applied.
+## 3. Choose a visualization 
 
 ![](/static/img/docs/panel-editor/select-visualization-8-0.png)
 
+When you choose a visualization, Grafana will offer visualization suggestions based on your query. You can select from the avaialble suggestions or you pick a visualization from the complete list of visualizations. Whether you pick one of the suggested visualizations or select one from the complete list of visualizations, Grafana will display a preview of your query results with that visualization applied.
+
```
Comment (achatterjee-grafana) [inline docs/sources/panels/add-a-panel.md:33]:
```suggestion
When you choose a visualization, Grafana will offer visualization suggestions based on your query. You can select from the available suggestions or you can pick a visualization from the complete list of supported visualizations. When you select a visualization, Grafana will show a preview of your query results with that visualization applied.
```

File: docs/sources/visualizations/_index.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ aliases = ["/docs/grafana/latest/panels/visualizations/"]
 
 Grafana offers a variety of visualizations to support different use cases. This section of the documentation highlights the built-in panels, their options and typical usage.
 
+> **Note:** If you are unsure which visualization to pick, Grafana can provide visualization suggestions based on the panel query. Additionally, Grafana will display a preview of the selected visualization type or visualization suggestion. Please see the [add a panel]({{< relref "../pnaels/add-a-panel.md" >}}) documentation for more info.
+
```
Comment (achatterjee-grafana) [inline docs/sources/visualizations/_index.md:12]:
```suggestion
> **Note:** If you are unsure which visualization to pick, Grafana can provide visualization suggestions based on the panel query. When you select a visualization, Grafana will show a preview with that visualization applied.  For more information, see the [add a panel]({{< relref "../pnaels/add-a-panel.md" >}}) documentation.

```

**Review by achatterjee-grafana — COMMENTED** [review]
Added minor doc edits. Thanks for adding the information!

**Comment by pkolyvas** [comment]
@achatterjee-grafana Thanks for the edits - I've commited them.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Pull request merged**